.vsts-pipelines/templates/default-build.yml

```diff
@@ -8,7 +8,7 @@
 #   poolName: string
 #       The name of the Azure DevOps agent pool to use.
 #   agentOs: string
-#       Used in templates to define variables which are OS specific. Typically from the set { Windows, Linux, macOS }
+#       Used in templates to define variables which are OS specific. Currently Windows only.
```

Comment:
This is good cleanup but I'm wondering whether it should be done in another PR. And, @mkArtakMSFT is it true we have no future plans for macOS or Linux builds of this repo❔

Reply:
It made setting up CodeQl easier as I only had to worry about windows jobs - The reason this pipeline had anything for non-win in the first place is because I copy/pasted it from another repo, it's never run a non-windows build since I added the azure pipeline